.github/workflows/split-packages.yml

```diff
@@ -0,0 +1,19 @@
+name: Split Packages
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  split-packages:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+
+    - name: Push split packages
+      run: make components-next
```

Comment:
Will the action runner have push access to all the split respositories?

Reply:
That's why I am not sure about :)  If not we'll have to fetch probably fetch the token for the user who triggered the action as pass it on / set as required env var.

Reply:
Yeah or create and use a github app that has repository push access to the split repos.